sdks/python/apache_beam/runners/worker/data_plane.py

```diff
@@ -168,7 +168,8 @@ def flush(self):
           self._large_flush_last_observed_timestamp = time.time()
           _LOGGER.warning(
               'Data output stream buffer size %s exceeds %s bytes. '
-              'This is likely due to a large element in a PCollection. '
+              'This is likely caused by a large element in a PCollection '
+              'that is passed between different fused stages. '
```

Comment:
might be too dataflow specific. `...that is passed between the SDK and a runner` might be better.